tests/namespaces.bats

```diff
@@ -63,58 +63,37 @@ load helpers
   # Check that with settings that don't require a user namespace, we can request to use a per-container network namespace.
   run_buildah run $RUNOPTS --net=container "$ctr" readlink /proc/self/ns/net
   if [[ $output == $mynetns ]]; then
-      expect_output "[output should not be '$mynetns']"
+      die "[/proc/self/ns/net (--net=container) should not be '$mynetns']"
   fi
 
   run_buildah run $RUNOPTS --net=private "$ctr" readlink /proc/self/ns/net
   if [[ $output == $mynetns ]]; then
-      expect_output "[output should not be '$mynetns']"
+      die "[/proc/self/ns/net (--net=private) should not be '$mynetns']"
   fi
 }
 
-idmapping_check_namespace() {
-  local _uidmapargs=$1
-  local _gidmapargs=$2
-  local _mynamespace=$3
-  local _output=$4
```

Comment:
This is now a localized function, defined within the appropriate `for` loop, because the uid/gid/mynamespace args really should not be args